x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/collapsed_item_actions.tsx

```diff
@@ -64,68 +65,70 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
       isOpen={isPopoverOpen}
       closePopover={() => setIsPopoverOpen(false)}
       ownFocus
+      panelPaddingSize="s"
       data-test-subj="collapsedItemActions"
     >
-      <EuiFormRow>
-        <EuiSwitch
-          name="enable"
-          disabled={!canSave}
-          checked={item.enabled}
-          data-test-subj="enableSwitch"
-          onChange={async () => {
-            if (item.enabled) {
-              await disableAlert(item);
-            } else {
-              await enableAlert(item);
+      <EuiContextMenuPanel>
```

Comment:
Not sure if this is new or caused by this PR but the first EuiContextMenuItem shows as highlighted for me (blue background).

<img width="193" alt="Screen Shot 2020-04-15 at 11 06 24 AM" src="https://user-images.githubusercontent.com/3694571/79355508-c124be80-7f0b-11ea-8fde-6218e42ed3c5.png">

Reply:
Set `hasFocus` prop to false in `EuiContextMenuPanel`